build-logic/src/main/groovy/io.micronaut.build.internal.sourcegen-module.gradle

```diff
@@ -0,0 +1,10 @@
+plugins {
+    id("io.micronaut.build.internal.sourcegen-base")
+    id("io.micronaut.build.internal.module")
+}
+
+micronautBuild {
+    binaryCompatibility {
+        enabled.set(false)
```

Comment:
why are we disabling binary compatibility?

Reply:
@sdelamo don't know, that's not my code. I didn't change this file, just move it to build-logic